### PR TITLE
HV: add Px Cx support for A3950 SoC

### PR DESCRIPTION
**Pull request by jsun26intel**
Intel Atom A3950 SoC also can be used in Acrn, and it has same Cx data as
Atom A3960 SoC.

Signed-off-by: Victor Sun <victor.sun@intel.com>
Reviewed-by: Kevin Tian <kevin.tian@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

File: hypervisor/arch/x86/cpu_state_tbl.c [hunk 1]
```diff
@@ -34,6 +34,23 @@ struct cpu_cx_data cx_a3960[] = {
 	{{SPACE_SYSTEM_IO, 0x8, 0, 0, 0x419}, 0x3, 0x96, 0x0A}  /* C3 */
 };
 
+/* The table includes cpu px info of Intel A3950 SoC */
+struct cpu_px_data px_a3950[] = {
```
Comment (dongyaozu) [inline hypervisor/arch/x86/cpu_state_tbl.c]: static?

File: hypervisor/arch/x86/cpu_state_tbl.c [hunk 2]
```diff
@@ -62,6 +79,10 @@ struct cpu_state_table {
 		{ARRAY_SIZE(px_a3960), px_a3960,
 		 ARRAY_SIZE(cx_a3960), cx_a3960}
 	},
+	{"Intel(R) Atom(TM) Processor A3950 @ 1.60GHz",
```
Comment (dongyaozu) [inline hypervisor/arch/x86/cpu_state_tbl.c]: Then you need to add 1.9G case too, right?

Comment (jsun26intel) [reply hypervisor/arch/x86/cpu_state_tbl.c]: 1.9G is already in, it is the first SoC we support when enable Pstate.

File: hypervisor/arch/x86/cpu_state_tbl.c (hunk 1 above)
Comment (jsun26intel) [reply hypervisor/arch/x86/cpu_state_tbl.c]: OK.
Can I keep it in this patch and submit another patch to change all data struct to "static const"?
We have other data defined without "static" in the file...

**Comment by jsun26intel** [comment]
@dongyaozu  I've updated push with static const change, please approve, thanks!

**Comment by lijinxia** [comment]
Verified+1

**Pull request merged**